org.fsnebula.Knossos.yml

```diff
@@ -0,0 +1,68 @@
+app-id: org.fsnebula.Knossos
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+  - org.freedesktop.Sdk.Extension.rust-stable
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+command: knossos
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --allow=bluetooth
+  - --share=network
+  - --filesystem=host
```

Comment:
this is a very broad filesystem permission

Reply:
In security terms it's no different from `home`, which already lets you tamper with `.profile` and various rc files, and any further-restricted permission would interfere unacceptably with typical use of the application. This is something I thought fairly carefully about before doing.